5-flexbox/assets/style.css

```diff
@@ -0,0 +1,51 @@
+article {
+    border: 2px solid black;
+    border-radius: 15px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 20px;
+    padding: 10px;
+}
+article img {
+    height: 100px;
+    object-fit: cover;
+    width: 100px;
+}
+body {
+    background-color: rgb(80, 158, 204);
+}
+div p {
+    margin-right: 20px;;
+    text-align: justify;
+    width: 40%;
+}
+h1,h2 {
+    display: flex;
+    justify-content: center;
```

Comment:
I know that you are using these properties for the exercise. FYI you can do the same effect with `text-align: center;`